### ISSUE TITLE
TPC-H data is split into two different plots

### ISSUE THREAD

**Issue by hendrikmakait** [coiled/benchmarks#1098]
#1044 caused the Dask-based TPC-H tests to have a different path. As a result, our reporting treats them as different tests. We should run a migration to adjust the data for the old entries. cc @milesgranger

**Comment by milesgranger** [comment]
They're about to change again in https://github.com/coiled/benchmarks/pull/1094

**Comment by mrocklin** [comment]
Yeah, I'm quite curious to learn more about the constraints of this repository that we need to adhere to.  There are ways I'd like to organize TPC-H but I don't know enough about the infrastructure of this repository to act with confidence.  What's the right way for me to learn more?  Should I be talking with someone in particular?

**Comment by fjetter** [comment]
> What's the right way for me to learn more?

Some of this is documented in https://github.com/coiled/benchmarks/blob/main/README.md but I guess the majority is currently 
the readme does only go into "deleting old data" and "changing the schema" but not about how to change test names.

> Should I be talking with someone in particular?

Either one of Hendrik, Guido or myself

**Comment by mrocklin** [comment]
> What's the right way for me to learn more?

I think I want more information than that.  There are some broad questions like "Should TPC-H even be part of benchmarks?" that I think would be good to talk through.  See for example https://github.com/coiled/benchmarks/pull/1094

**Comment by mrocklin** [comment]
>> Should I be talking with someone in particular?

> Either one of Hendrik, Guido or myself

Should I schedule a meeting with all three of you, or just one?

**Comment by fjetter** [comment]
> Should I schedule a meeting with all three of you, or just one?

I suggest to keep it small. I'm around now if you are available
